workflow-bot-app/build.gradle

```diff
@@ -82,3 +84,9 @@ publishing {
         }
     }
 }
+
+processResources {
```

Comment:
actually


processResources {
    filesMatching('application.yaml') {
        expand(project.properties)
    }
}

could be used to also set:


bdk:
  default-headers:
    User-Agent: "Symphony-WDK/${project.version} Java/\${java.version}"

in the default application yaml

Reply:
our yours is simpler to make it work, no need to escape